src/getCoinCombination.test.js

```diff
@@ -7,4 +7,20 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return number of penny', () => {
```

Comment:
Which number and when should it return this number?
We need a clear test name. Consider it your test case summary. Here is a good example:
'should return '1' pennie when '1' is entered'